### ISSUE TITLE
Plugin Setup - "Mask"

### ISSUE THREAD

**Issue by xXReDxSparoweXx** [ccseer/Seer#73]
How do I configure the "Mask" plugin or See itself for files with a ".exe" extension? Whenever I open some ".exe" files, namely "Adobe Photoshop", Seer will activate the quick look window just as it does for other files. Only it does this on its own without me wanting it to or by executing the "spacebar" hotkey. This isn't a "big" issue, just more of an annoyance with a program that I utilize just about as much as I do "Chrome" or any other windows programs. It's also a great program which should've been included with Windows OS from the get go IMHO. I saw that there was another issue related to Photoshop and Seer in the issue tracker, so I can assume it's being looked into? 

TLDR: How do I configure the "Mask" plugin to mask ".exe" files from Seer so that I don't have it popping up every time I use Photoshop and all other ".exe" files? I honestly don't know why Seer has this as a feature? I could be wrong but I just see no point in wanting to utilize Seer for viewing ".exe" files.

**Comment by ccseer** [comment]
I think macOS does the same thing  for viewing executable files.
